pmd-cataloger/src/main/java/sfdc/sfdx/scanner/pmd/Main.java

```diff
@@ -18,8 +20,20 @@ public static void main(String[] args) {
     String pmdPath = args[0];
     String pmdVersion = args[1];
     List<String> supportedLangs = new ArrayList<>(Arrays.asList(args[2].split(",")));
+    String encodedCustomClasspathMapping = args[3];
 
-    PmdRuleCataloger prc = new PmdRuleCataloger(pmdVersion, pmdPath, supportedLangs);
+    // Decode customClasspathMap into a JSOn string
+    final String customClasspathMapping = decodeCustomClasspathMapping(encodedCustomClasspathMapping);
```

Comment:
Passing language to resource mapping as an encoded JSON and decoding it before passing the string to PmdRuleCataloger.java